shared/folders/row.native.js

```diff
@@ -0,0 +1,193 @@
+// @flow
+import React from 'react'
+import {Image} from 'react-native'
+import type {Folder} from './list'
+import {Box, Text, Icon, Avatar, Meta} from '../common-adapters'
+import type {Props as IconProps} from '../common-adapters/icon'
+import {globalStyles, globalColors} from '../styles/style-guide'
+
+const Avatars = ({styles, users, isPublic}) => {
+  const contents = users.length === 1 || users.length === 2
+      ? <Avatar size={32} username={users[users.length - 1].username} />
+      : <Icon type={styles.groupIcon} />
+
+  if (isPublic) {
+    return <Box style={styles.avatarContainer}>{contents}</Box>
+  }
+
+  // $FlowIssue doesn't like images
```

Comment:
flow tries to resolve this like a js file so it gets confused, dunno if ther's some larger workaround for this